src/workloads/query/multiplanner/BlockingPlanWin.yml

```diff
@@ -0,0 +1,219 @@
+        x28: *distribution,
+        x29: *distribution,
+        x30: *distribution,
+        x31: *distribution,
+        x32: *distribution,
+        x33: *distribution,
+        x34: *distribution,
+        x35: *distribution,
+        x36: *distribution,
+        x37: *distribution,
+        x38: *distribution,
+        x39: *distribution,
+        x40: *distribution,
+        x41: *distribution,
+        x42: *distribution,
+        x43: *distribution,
+        x44: *distribution,
+        x45: *distribution,
+        x46: *distribution,
+        x47: *distribution,
+        x48: *distribution,
+        x49: *distribution,
+        x50: *distribution,
+        x51: *distribution,
+        x52: *distribution,
+        x53: *distribution,
+        x54: *distribution,
+        x55: *distribution,
+        x56: *distribution,
+        x57: *distribution,
+        x58: *distribution,
+        x59: *distribution,
+        x60: *distribution,
+        x61: *distribution,
+        x62: *distribution,
+        x63: *distribution
+      }
+      indexes:
+      - keys: {x1: 1}
+        options: {name: index1}
+      - keys: {x2: 1}
+        options: {name: index2}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: MultiplannerPicksBlockingPlanSelectivity0.01
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: QueryTemplate
+    Parameters:
+      active: [3]
+      nopInPhasesUpTo: *maxPhase
+      repeat: *queryRepeats
+      database: *db
+      collection: *coll
+      query: {
+        Filter: {x1: {$lt: 0.01}},
+        Options: {Sort: '{"x2": {"$numberInt": "1"}}'}
+      }
+
+- Name: MultiplannerPicksBlockingPlanSelectivity0.1
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: QueryTemplate
+    Parameters:
+      active: [4]
+      nopInPhasesUpTo: *maxPhase
+      repeat: *queryRepeats
+      database: *db
+      collection: *coll
+      query: {
+        Filter: {x1: {$lt: 0.1}},
+        Options: {Sort: '{"x2": {"$numberInt": "1"}}'}
+      }
+
+- Name: MultiplannerPicksBlockingPlanSelectivity0.5
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: QueryTemplate
+    Parameters:
+      active: [5]
+      nopInPhasesUpTo: *maxPhase
+      repeat: *queryRepeats
+      database: *db
+      collection: *coll
+      query: {
+        Filter: {x1: {$lt: 0.5}},
+        Options: {Sort: '{"x2": {"$numberInt": "1"}}'}
+      }
+
+- Name: MultiplannerPicksBlockingPlanSelectivity0.9
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: QueryTemplate
+    Parameters:
+      active: [6]
+      nopInPhasesUpTo: *maxPhase
+      repeat: *queryRepeats
+      database: *db
+      collection: *coll
+      query: {
+        Filter: {x1: {$lt: 0.9}},
+        Options: {Sort: '{"x2": {"$numberInt": "1"}}'}
```

Comment:
Are you sure multi planner will pick a blocking plan here?

Reply:
Same as above